src/test/java/com/investmenttracker/enterprise/EnterpriseApplicationTests.java

```diff
@@ -18,8 +18,8 @@
 @SpringBootTest
 class EnterpriseApplicationTests {
 	@Autowired
-	IInvestmentService investmentService;
-	investment Investment = new investment();
+	private IInvestmentService investmentService;
+	private Investment investment = new Investment();
```

Comment:
Per best practices, you should use access modifiers on variables. https://www.tutorialspoint.com/java/java_modifier_types.htm